client/signup/steps/domains/index.jsx

```diff
@@ -145,6 +148,34 @@ class DomainsStep extends Component {
 		return isPlansStepExistsInFutureOfFlow && ! isPlanStepSkipped;
 	};
 
+	componentDidMount() {
+		const signupFlows = [
+			'onboarding',
+			'free',
+			'personal',
+			'premium',
+			'business',
+			'ecommerce',
+			'with-theme',
+			'personal-monthly',
+			'premium-monthly',
+			'business-monthly',
+			'ecommerce-monthly',
+			'with-design-picker',
```

Comment:
Is there a `with-design-picker` flow?

Reply:
I got that from a list that the marketing analysts use for reporting, but I wouldn't be surprised if it's an outdated list. I removed it from the PR.